CMakeLists.txt

```diff
@@ -220,11 +220,13 @@ if(NOT AUTOWIRING_IS_EMBEDDED)
   )
 
   # Install autoboost headers
-  install(
-    DIRECTORY ${PROJECT_SOURCE_DIR}/contrib/autoboost/autoboost
-    DESTINATION include
-    COMPONENT autowiring
-  )
+  if(NOT autowiring_USE_LIBCXX)
```

Comment:
No, because autowiring_USE_LIBCXX is presently set to "on" on all platforms, except on Mac, where it might be set to "off".